src/components/inputs/radio.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+    <div class="scbd-common radio  form-check form-check-inline">   
```

Comment:
only add inline class when the attribute `inline` exists 

`:class="{'form-check-inline' : $attrs.inline}"`